internal/apiutil/auth.go

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Mainflux
```

Comment:
Consider renaming `apiutil` to something closer to what this package is for - maybe `initutil` or just `init`.